app/presenters/blacklight/thumbnail_presenter.rb

```diff
@@ -72,7 +72,7 @@ def default_thumbnail_value(image_options)
     end
 
     def thumbnail_value_from_document
-      Array(thumbnail_field).lazy.map { |field| retrieve_values(field_config(field)).first }.reject(&:blank?).first
+      Array(thumbnail_field).lazy.map { |field| retrieve_values(field_config(field)).first }.compact_blank.first
```

Comment:
@jcoyne looks like this line causes an issue with rails 6.  Can you reasonably bump the rails 6 version to 6.1.3.1 to grab it? (https://apidock.com/rails/Enumerable/compact_blank) or disable the rubocop check and go back to the original code?

Reply:
Yeah, according to https://github.com/rspec/rspec-rails#rspec-rails-- we should be using version 5.x if we need to support Rails 6.0